etc/cc-dev.conf

```diff
@@ -49,9 +41,26 @@ server {
     proxy_pass         http://localhost:7782/connect;
     proxy_redirect     off;
 
-    proxy_set_header   Host             $host;
-    proxy_set_header   X-Real-IP        $remote_addr;
-    proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
+    proxy_max_temp_file_size 0;
+
+    client_max_body_size       10m;
+    client_body_buffer_size    128k;
+
+    proxy_connect_timeout      90;
```

Comment:
* Isn't 90s kind of long?  [The manual](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_connect_timeout) says "It should be noted that this timeout cannot usually exceed 75 seconds."  The default is 60s; is there any reason to increase it?  (I'd probably reduce it to 10s or something, so users get a nice nginx-generated error page instead of just waiting.

Reply:
Done.